tshirts/TshirtsTest.java

```diff
@@ -0,0 +1,15 @@
+package tshirts;
+
+public class TshirtsTest {
+
+  
+
+  public void sizeCheck() {
+    assertTrue(Tshirts.size(37) == "S");
```

Comment:
Test cases need to be strengthened here to cover all boundary conditions and values before and after.